pkg/apis/sources/v1alpha1/register.go

```diff
@@ -20,11 +20,16 @@ limitations under the License.
 // +k8s:openapi-gen=true
 // +k8s:deepcopy-gen=package,register
 // +k8s:conversion-gen=github.com/knative/eventing-sources/pkg/apis/sources
+// +genclient
```

Comment:
Don't the `+genclient` and `+k8s:deepcopy-gen` directives normally go above the type structs? Why move them here?

See https://github.com/kubernetes-sigs/kubebuilder/blob/804f002125094de37d15e758eec54e751843509a/test/project/pkg/apis/ship/v1beta1/register.go#L20-L24 and https://github.com/kubernetes-sigs/kubebuilder/blob/804f002125094de37d15e758eec54e751843509a/test/project/pkg/apis/ship/v1beta1/frigate_types.go#L38-L39

Reply:
ah yes thanks.